code/script.js

```diff
@@ -1,325 +1,1505 @@
+    mainRole: "support",
+    accessories: [],
+  },
+  {
+    name: "Zoe",
+    img: "images/RiotX_ChampionList_zoe.jpg",
+    hair: "orange",
+    homeRegion: "targon",
+    mainRole: "mage",
+    accessories: [],
+  },
+  {
+    name: "Zyra",
+    img: "images/RiotX_ChampionList_zyra.jpg",
+    hair: "red",
+    homeRegion: "ixtal",
+    mainRole: "mage",
+    accessories: [],
+  },
+];
 
 // Global variables
-let secret
-let currentQuestion
-let charactersInPlay
+let secret,
+  currentQuestion,
+  charactersInPlay,
+  championToCheck,
+  startTime,
+  endTime;
+let questionsAsked = 0;
 
-// Draw the game board
+// Function that draws the game board
 const generateBoard = () => {
-  board.innerHTML = ''
-  charactersInPlay.forEach((person) => {
+  board.innerHTML = ``;
+  charactersInPlay.forEach((champion) => {
     board.innerHTML += `
-      <div class="card">
-        <p>${person.name}</p>
-        <img src=${person.img} alt=${person.name}>
+      <div class="card" style="background-image: url('${champion.img}');background-size: cover;">
+        <p>${champion.name}</p>
         <div class="guess">
-          <span>Guess on ${person.name}?</span>
-          <button class="filled-button small" onclick="guess('${person.name}')">Guess</button>
+          <span>Guess on ${champion.name}?</span>
+          <button class="filled-button small" onclick="guess('${champion.name}')">Guess</button>
         </div>
       </div>
-    `
-  })
-}
+    `;
+  });
+};
 
-// Randomly select a person from the characters array and set as the value of the variable called secret
+// Function that randomly select a champion from the characters array and set as the value of the variable called secret
 const setSecret = () => {
-  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
-}
+  secret =
+    charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)];
+};
 
-// This function to start (and restart) the game
+// A functions which starts and restarts the game, start the game, change and resets some data so the game is loaded from the beginning
+// if you pressed the restart butten. Invokes a lot of functions.
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
-}
+  startTimer();
+  questionsAsked = 0;
+  questionsHistory.innerHTML = ``;
+  guessCounter.innerHTML = `Questions asked: ${questionsAsked}`;
+  winOrLose.classList.remove("active");
+  document.body.style.overflow = "auto";
+  charactersInPlay = CHARACTERS;
+  setCharacterInPlayCounter();
+  generateBoard();
+  setSecret();
+  selectQuestion();
+};
 
-// setting the currentQuestion object when you select something in the dropdown
-const selectQuestion = () => {
-  const category = questions.options[questions.selectedIndex].parentNode.label
+// Functions that start and ends a timer.
+const startTimer = () => {
```

Comment:
it doesn't look that difficult to incorporate a timecounter, now when I have seen your way to do so =) You make the code look so easy and "obvious".